File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int left=0,right=numbers.size()-1;
        vector<int>rett;
        while(left<right){
            if(numbers[left]+numbers[right]>target){
                right--;
            }else if (numbers[left]+numbers[right]<target){
                left++;
            }else{
                rett.push_back(left+1);
                rett.push_back(right+1);
                break;
            }
        }
        return rett;
    }
};

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        left,right= 0,len(numbers)-1
        rett=[]
        while (left<=right):
            if (numbers[left]+numbers[right]>target):
                right-=1
            elif (numbers[right]+numbers[left]<target):
                left+=1
            else :
                rett.append(left+1)
                rett.append(right+1)
                break
        return rett